### PR TITLE
Improve issue list filter

### PR DESCRIPTION
**Pull request by wxiaoguang**
Partial regression of #24393, not only regression, but broken for long time, 24393 didn't really improve it but used wrong `overflow: scroll`.

Actually, that "ui secondary filter menu labels" shouldn't be set as scrollable (I missed that at that time), the problem is: if a "ui menu" has "dropdown" items, then it should not be scrollable. Otherwise the dropdown menu can't be shown correctly.

And there are more problems:

* The "issue-filters" shouldn't be used anywhere else (copying&pasting problem again ....)
* There is also an "issue-actions" container, it should also be fixed.
* There are similar problems on the milestone page.
* The old comment in code: "grid column" doesn't work well.

The major changes of this PR are: use "flex: 1" instead of "ui grid column".

After this PR, not 100% perfect but much better than before.

# Before

image

image

# After

image

image

image

----

image

image

image

### CODE REVIEW

**Comment by silverwind** [comment]
image

Why is there a scrollbar when content only scrolls so little? Looks like a overflow bug that we could may as well fix here.

**Comment by wxiaoguang** [comment]
> Why is there a scrollbar when content only scrolls so little? Looks like a overflow bug that we could may as well fix here.

It's not a bug. Because the "menu" has max-height: 500px, and these labels happen to have a total height slightly larger than 500px.


If I delete a label from backend, then there is no scrollbar:

image

**Comment by silverwind** [comment]
I see. BTW these label dropdowns don't look so pretty but that's a topic for another PR.

File: web_src/css/repository/issue-list.css [hunk 1]
```diff
@@ -0,0 +1,14 @@
+.issue-list-toolbar {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+  gap: 1em;
```
Comment (silverwind) [inline web_src/css/repository/issue-list.css]:
```suggestion
  gap: 1rem;
```

**Review by silverwind — APPROVED** [review]

**Review by lunny — APPROVED** [review]

**Pull request merged**